packages/storefront/src/pages/utilities/introduction.md

```diff
@@ -4,6 +4,12 @@
 
 ## Porsche Design System - Utilities
 
+<p-inline-notification heading="Important note" state="error" persistent="true">
+  The <code>Porsche Design System - Utilities</code> package is <strong>deprecated</strong> and will no longer be maintained.<br>
+  All Porsche Design System utilities are now provided at @porsche-design-system/components-{js|angular|react|vue}/styles in from of styles and Design Tokens.<br>
+  To make the migration easier have a look at the <a href="news/changelog/utilities">changelog</a> which contains a migration path.
+</p-inline-notification>
+
 This package contains helpful SCSS & JS variables, functions and helpers.
 
 ### Install
```

Comment:
Perhaps adjust next section to mention npmjs?

Reply:
Could you elaborate? I do not fully understand what you mean.